services/docker-compose.devel.yml

```diff
@@ -18,18 +18,22 @@ services:
   webserver:
     image: services_webserver:dev
     build:
+      args:
+        - HOST_GID_ARG=${HOST_GID:?Undefined host gid}
```

Comment:
What does this syntax mean?

Reply:
RTFM? :-b

See the end of the section https://docs.docker.com/compose/compose-file/#variable-substitution